testing/burp/burp.init

```diff
@@ -27,5 +27,5 @@ reload() {
 }
 
 mon() {
-	$command -c "$cfgfile" -a S
+	$command -a S
```

Comment:
Because it has to use the default configuration. With 2.1.x this is the only correct way to call the status page. Older versions needed the config file.